hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -161,6 +161,9 @@ public class SingleTransactionRecordBuilderImpl
     private List<AbstractMap.SimpleEntry<ContractActions, Boolean>> contractActions = new LinkedList<>();
     private List<AbstractMap.SimpleEntry<ContractBytecode, Boolean>> contractBytecodes = new LinkedList<>();
 
+    @Nullable
+    private Long signerNonce = null;
```

Comment:
Where is this field used? Doesn't it need to be set on an underlying PBJ `TransactionRecord.Builder` somewhere?

Reply:
You are right, this field isn't really needed.